pkgs/by-name/lu/lumafly/package.nix

```
@@ -0,0 +1,73 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildDotnetModule,
+  dotnetCorePackages,
+  zlib,
+  icu,
+  fontconfig,
+  openssl,
+  xorg,
+  icoutils,
+  copyDesktopItems,
+  makeDesktopItem,
+}:
+buildDotnetModule rec {
+  pname = "lumafly";
+  version = "3.2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "TheMulhima";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-oDSM5Ev9SCjbvCgDZcpzm2bVnzG04yy/WaSwJyh0b18=";
+  };
+
+  projectFile = "Lumafly/Lumafly.csproj";
+
+  nugetDeps = ./deps.nix;
+
+  dotnet-sdk = dotnetCorePackages.sdk_7_0;
+
+  selfContainedBuild = true;
+
+  runtimeDeps = [
+    zlib
+    icu
+    fontconfig
+    openssl
+    xorg.libX11
+    xorg.libICE
+    xorg.libSM
+  ];
+
+  nativeBuildInputs = [
+    icoutils
+    copyDesktopItems
+  ];
+
+  postFixup = ''
+    # Icon for the desktop file
+    icotool -x $src/Lumafly/Assets/Lumafly.ico
+    install -D Lumafly_1_32x32x32.png $out/share/icons/hicolor/32x32/apps/lumafly.png
+  '';
+
+  desktopItems = [(makeDesktopItem {
+    desktopName = "Lumafly";
+    name = "lumafly";
+    exec = "Lumafly";
+    icon = "lumafly";
+    comment = meta.description;
+    type = "Application";
+    categories = [ "Game" ];
+  })];
+
+  meta = with lib; {
```

```suggestion
  meta = {
```

https://github.com/NixOS/nixpkgs/issues/208242